CHANGELOG.md

```diff
@@ -2,13 +2,15 @@
 Notable updates:
 
 ### Added
+* Add support for benchmarking all Forest db backends
```

Comment:
The benchmarking script isn't part of our release and shouldn't be added to the changelog. Same goes for the entry under `Fixed`.

Reply:
Oh you are right. So will remove them.